ecs-cli/modules/clients/aws/cloudformation/cluster_template.go

```diff
@@ -602,6 +602,19 @@ var cluster_template = `
       "Condition": "CreateEcsInstanceRole",
       "Type": "AWS::IAM::Role",
       "Properties": {
+        "Policies": [
+           {
+             "PolicyName": "enable-container-instance-tagging",
```

Comment:
we should name this policy the same way we name the role and `registry-creds` policy. 
e.g., "amazon-ecs-cli-setup-<cluster_name>-instance-tagging-policy"